### PR TITLE
Fix bug that imported proto files didn't be generated

### PR DESCRIPTION
**Pull request by ridewindx**

### CODE REVIEW

**Comment by ridewindx** [comment]
Path of the imported proto files should be specified by flag "-I", and then protoc-plugin will find them and generate typescript files for them. So file "wkt.meta.json" is also not needed any more.

**Comment by smnbbrv** [comment]
Hi @ridewindx 

thank you for PR.

What kind of issue are you trying to fix? Is something not getting generated? Could you please provide an example?

**Comment by ridewindx** [comment]
For example after applied this PR:
```
proto/
|-- captpb
|   |-- api.pb.ts
|   |-- api.pbconf.ts
|   `-- api.pbsc.ts
|-- gogoproto
|   `-- gogo.pb.ts
|-- google
|   |-- api
|   |   |-- annotations.pb.ts
|   |   `-- http.pb.ts
|   `-- protobuf
|       |-- descriptor.pb.ts
|       |-- duration.pb.ts
|       |-- empty.pb.ts
|       `-- timestamp.pb.ts
`-- validate
    `-- validate.pb.ts
```

The main proto file is captpb/api.proto, but it imports some thirdparty proto files. However, `protoc-gen-ng` did not generate those typescript files for them. And including `wkt.meta.json` leads to generate all google/protobuf files, but that is obviously unnecessary.

File: src/main.ts [hunk 1]
```diff
@@ -12,14 +12,13 @@ import { PbscFile } from './output/files/pbsc-file';
...
 import { Services } from './services';
-import wkt from './wkt.meta.json';
 
 function main() {
   CodeGeneratorRequest()
     .then(r => {
       const protocInput = r.toObject()
-      const filesToGenerate = protocInput.protoFileList.filter(p => protocInput.fileToGenerateList.indexOf(p.name) !== -1);
-      const protos = [...(wkt as unknown as Proto[]), ...filesToGenerate].map(proto => new Proto(proto));
+      const filesToGenerate = protocInput.protoFileList;
```
Comment (smnbbrv) [inline src/main.ts]: Please turn into one-liner

File: src/main.ts [hunk 2]
```diff
@@ -12,14 +12,13 @@ import { PbscFile } from './output/files/pbsc-file';
 import { PbwscFile } from './output/files/pbwsc-file';
 import { Printer } from './output/misc/printer';
 import { Services } from './services';
-import wkt from './wkt.meta.json';
```
Comment (smnbbrv) [inline src/main.ts]: Please also remove the wkt.meta.json file as well as long as it is not necessary any longer

**Comment by smnbbrv** [comment]
Hi @ridewindx 

thanks for clarification. I checked the patch and the output looks valid (and better).

Could you please check the PR comments and change your commit message to a valid semantic-release commit message?

E.g. `fix: generate all imported proto files` or the message you like.

And, please change the PR target to `develop` branch

**Comment by ridewindx** [comment]
@smnbbrv OK

**Pull request closed without merging**